README.md

```diff
@@ -21,6 +21,12 @@ efficiency, SCMs can easily be coupled to other models and used to
 design scenarios, emulate more complex climate models, and conduct
 uncertainty analyses.
 
+    #> Warning: Using `size` aesthetic for lines was deprecated in ggplot2 3.4.0.
+    #> ℹ Please use `linewidth` instead.
+    #> This warning is displayed once every 8 hours.
+    #> Call `lifecycle::last_lifecycle_warnings()` to see where this warning was
+    #> generated.
+
```

Comment:
We don't want to commit this--please remove! It looks like this is coming from line 39 in the Rmd file, so change that to
```
    geom_line(aes(year, value, color = scenario), linewidth= 1) +
```

Reply:
nice catch! Thanks!